### PR TITLE
Refresh Rancher Desktop install of Epinio

### PR DESCRIPTION
**Pull request by jhkrug**
Needs some technical review and work through.

Version numbers mentioned are old. Need updating. Does the technical content still apply?, etc.

RD has a push-button Epinio install which works, kinda.

### CODE REVIEW

File: docs/installation/other_inst_scenarios/install_epinio_on_rancher_desktop.md [hunk 1]
```diff
@@ -1,13 +1,16 @@
 ---
-sidebar_label: "Installing Epinio On Rancher Desktop (local)"
+sidebar_label: "Installing Epinio on Rancher Desktop"
 sidebar_position: 1
-title: ""
+title: "Installing Epinio on Rancher Desktop"
+description: iHow o install Epinio using a local Rancher Desktop installation.
```
Comment (andreas-kupries) [inline docs/installation/other_inst_scenarios/install_epinio_on_rancher_desktop.md:5]:
```suggestion
description: How to install Epinio using a local Rancher Desktop installation.
```

**Review by andreas-kupries — CHANGES_REQUESTED** [review]
Change requested is only for 2 typos.
Unfortunately I have no answers for the technical questions.

File: docs/installation/other_inst_scenarios/install_epinio_on_rancher_desktop.md [hunk 2]
```diff
@@ -39,6 +42,7 @@ Rancher Desktop configures its own load-balancer to expose Traefik on `127.0.0.1
 
 The Epinio installation is similar on Linux, MacOS and Windows:
 
+<!--TODO: Any particular reason why the Windows sheel prefers the latest?-->
```
Comment (andreas-kupries) [inline docs/installation/other_inst_scenarios/install_epinio_on_rancher_desktop.md:45]: Looking through the history this part was written by @ldevulder .

File: docs/installation/other_inst_scenarios/install_epinio_on_rancher_desktop.md [hunk 3]
```diff
@@ -62,6 +66,13 @@ epinio login -u admin https://epinio.127.0.0.1.sslip.io
 
 For RancherDesktop on Linux, in order to be able to open ports from `443` (and above), in order to access the URL set in `global.domain` (i.e. 127.0.0.1.sslip.io), you need to set the start port of the `unprivileged` list to a lower number:
 
+<!--
+TODO:Does this need to be 80 as per the RD installation docs?
```
Comment (andreas-kupries) [inline docs/installation/other_inst_scenarios/install_epinio_on_rancher_desktop.md:70]: Looking through the history this part came from @nunix .

File: docs/installation/other_inst_scenarios/install_epinio_on_rancher_desktop.md (hunk 2 above)
Comment (jhkrug) [reply docs/installation/other_inst_scenarios/install_epinio_on_rancher_desktop.md]: Thanks.

File: docs/installation/other_inst_scenarios/install_epinio_on_rancher_desktop.md (hunk 3 above)
Comment (jhkrug) [reply docs/installation/other_inst_scenarios/install_epinio_on_rancher_desktop.md]: Thanks.

**Review by andreas-kupries — APPROVED** [review]

**Pull request merged**